### DISCUSSION TITLE
Monika no longer asks for a goodnight kiss...?

### DISCUSSION THREAD

**Discussion by Hhhhhhjjjjkkk** [Monika-After-Story/MonikaModDev#8617] [QA - tech support + general] [answered]
Uh so this is strange, normally monika asks for a goodnight kiss if the player goes to bed from 9 pm to 10PM (maybe im wrong) yea, before she always asked me for a goodnight kiss but for like 2 days she just stopped and just hits me with the "okay (name)"... did i do something wrong...? my affection is nearly at 500 now is this a bug or something?

I thought it was a submod fault so i decided to download ddlc and mas again and put them in a new folder, i opened the game said "im going to sleep" then she asked for a goodnight kiss but then i tried again, after spending some time with her well roughly 5 mins and we back to "okay (name) sweet dreams) like dude what am i doing wrong

**Comment by multimokia** [comment, marked as answer]
It's a random chance for her to do it, not guaranteed, you just got pretty lucky

**Comment by Hhhhhhjjjjkkk** [comment]
> It's a random chance for her to do it, not guaranteed, you just got pretty lucky

sure its random chance? cos like ive tried full on spamming the "im going to sleep" and apart from the time i got it when i redownloaded the game, i didnt get it ONCE im hoping this is nothing bad or like the chances of her asking for a goodnight kiss are REALLY low?

**Comment by SpaceyFlakes** [comment]
Like multimokia said, it's random chance. My Monika rarely gives me a goodnight kiss so it's always a nice surprise to get one. You're just really lucky dude!
